test/integration/api-schema/_utils.js

```diff
@@ -5,19 +5,20 @@ const fs = require('fs')
 const tmpdir = require('os').tmpdir
 const join = require('path').join
 
-const validator = require('is-my-json-valid')
-const refParser = require('@apidevtools/json-schema-ref-parser')
+const Ajv = require('ajv')
 const rimraf = require('rimraf')
 const thunky = require('thunky')
 
+const ajv = new Ajv({ allErrors: true })
+
 const schemaDir = thunky(function (cb) {
-  const dir = join(tmpdir(), '.schemacache')
+  const dir = join(tmpdir(), 'apm-schema-v2-cache')
   fs.stat(dir, function (err) {
```

Comment:
Just noting the change in directory name from `.schemacache` to `apm-schema-v2-cache`.  👍 to a more specific caching name.